index.html

```diff
@@ -6,17 +6,12 @@
   <meta charset="UTF-8">
   <title>Fimento API</title>
   <link rel="icon" type="image/png" href="./dist/assets/favicon.png">
-  <link rel="stylesheet" type="text/css" href="./dist/css/global.css" />
+  <!-- <link rel="stylesheet" type="text/css" href="./dist/css/global.css" /> -->
```

Comment:
Any reason we can't just remove  this?

Reply:
I just accidently hid it but I restored it in my last commit